protocol/test/forge/v2/templeLineOfCredit/TlcLiquidations.t.sol

```diff
@@ -66,6 +66,27 @@ contract TempleLineOfCreditTestCheckLiquidity is TlcBaseTest {
 contract TempleLineOfCreditTestBatchLiquidate is TlcBaseTest {
     event StrategyCreditAndDebtBalance(address indexed strategy, address indexed token, uint256 credit, uint256 debt);
 
+    function test_batchLiquidate_paused() external {
```

Comment:
this test satisfies the functionality. what test i think is missing is the full scenario. After rescue mode enabled, liquidation is paused, `x` amount of time passes, liquidation unpaused, users can repay. You also mentioned that the rate would be set to 0 as soon as rescue mode is enabled. I think a complete test for this scenario would help

Reply:
Added in https://github.com/TempleDAO/temple/pull/842/commits/d78b013f9ba2da7a2e131ae2c5e0ceeb2cb59b90